sos/report/plugins/scsi.py

```diff
@@ -61,14 +61,13 @@ def setup(self):
         ])
 
         scsi_hosts = glob("/sys/class/scsi_host/*")
-        self.add_blockdev_cmd("udevadm info -a %(dev)s", devices=scsi_hosts,
-                              prepend_path='/sys/class/scsi_host')
+        self.add_device_cmd("udevadm info -a %(dev)s", devices=scsi_hosts)
```

Comment:
I dont have means to test this, but I believe the `prepend_path` option is really not further needed.

Reply:
At the moment, correct. I'm a bit hesitant to remove it because I think there may be a use for it in the future, potentially (e.g. a container-focused sos report that may need some host-level collections as well), but I don't know yet how practical that actually is or if it's entirely irrelevant.

I'd rather leave it for now and remove it after awhile of no usage than pulling it out now and having to replace it later.